core/src/test/resources/importSingleLine/ImportSingleLine.source

```diff
@@ -0,0 +1,87 @@
+40 columns                              |
+<<< One per line
+import foo.bar
+
+import kaz.bar
+object a
+>>>
+import foo.bar
+
+import kaz.bar
+object a
+<<< One per line, with docstring
```

Comment:
Are these tests necessary for the new flag? I would try to add only tests that check the functionality of the new flag.